### PR TITLE
{devel,lib,tools}[fosscuda/2019b] Bazel v3.7.2, Horovod v0.22.1, TensorFlow v2.5.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by Flamefire**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
> Test report by @boegelbot
> **FAILED**
> Build succeeded for 1 out of 3 (3 easyconfigs in total)
> cnx1 - Linux rocky linux 8.4, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz (haswell), Python 3.6.8
> See https://gist.github.com/9cb56c940b0197cd4c7c65834e056d13 for a full test report.

I am not sure if the `fosscuda` version will run on generoso.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in total)
taurusml28 - Linux RHEL 7.6, POWER, 8335-GTX (power9le), Python 2.7.5
See https://gist.github.com/70ca16295a45b3b14acef3d668c9cf40 for a full test report.

File: easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+checksums = [
+    'de255bb42163a915312df9f4b86e5b874b46d9e8d4b72604b5123c3a845ed9b1',  # bazel-3.7.2-dist.zip
+    'f87ad8ad6922fd9c974381ea22b7b0e6502ccad5e532145f179b80d5599e24ac',  # Bazel-3.4.1-fix-grpc-protoc.patch
+    '8706ecc99b658e0a96c38dc2c23e44da35059b85f308602aac76a6d6680376e7',  # Bazel-3.7.1_fix-protobuf-env.patch
+]
+
+builddependencies = [
+    ('binutils', '2.32'),
+    ('Python', '3.7.4'),
+    ('Zip', '3.0'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/b/Bazel/Bazel-3.7.2-GCCcore-8.3.0.eb:25]:
```suggestion
    ('Zip', '3.0'),
    ('UnZip', '6.0'),
```

File: easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.0-fosscuda-2019b-Python-3.7.4.eb [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+toolchain = {'name': 'fosscuda', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Bazel', '3.7.2'),
+    ('protobuf', '3.10.0'),
+    # git 2.x required, see also https://github.com/tensorflow/tensorflow/issues/29053
+    ('git', '2.23.0', '-nodocs'),
+    ('pybind11', '2.4.3', versionsuffix),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.5.0-fosscuda-2019b-Python-3.7.4.eb:18]:
```suggestion
    ('pybind11', '2.4.3', versionsuffix),
    ('UnZip', '6.0'),
```

**Comment by Flamefire** [comment]
Test report by @Flamefire
**FAILED**
Build succeeded for 1 out of 3 (3 easyconfigs in total)
taurusa8 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz (broadwell), Python 2.7.5
See https://gist.github.com/e7f8d13f9d9e82c6591ca0d7bafa9f31 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
jsfc061 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/c3eb6f7b1ff95990ed1e10c8c493feb8 for a full test report.

**Review by SebastianAchilles — DISMISSED** [review]

**Comment by terjekv** [comment]
Test report by @terjekv
**FAILED**
Build succeeded for 5 out of 8 (4 easyconfigs in total)
ninhursaga.uio.no - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Silver 4216 CPU @ 2.10GHz (cascadelake), Python 3.6.8
See https://gist.github.com/05d3133d94df04e1f1192e68fa3200fb for a full test report.

**Comment by terjekv** [comment]
Test report by @terjekv
**FAILED**
Build succeeded for 5 out of 8 (4 easyconfigs in total)
nam-shub-02.uio.no - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6226R CPU @ 2.90GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d8b4172792f20d081d11c9941d9f55dc for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
taurusa5 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz (broadwell), Python 2.7.5
See https://gist.github.com/a38f98db5bf7c42ec3fba97103e8bb52 for a full test report.

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 6 out of 6 (4 easyconfigs in total)
taurusml16 - Linux RHEL 7.6, POWER, 8335-GTX (power9le), Python 2.7.5
See https://gist.github.com/78fecffb9d96b74ab5c5096f44466176 for a full test report.

**Comment by Flamefire** [comment]
@terjekv CC 8.x is not supported by this NVCC

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
jsfc060 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/19dcc36b98725934110e33d1f2e87020 for a full test report.

**Comment by Flamefire** [comment]
@SebastianAchilles Fixed the tests. Build ok, but failed to upload. Restarting

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
taurusa4 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2603 v4 @ 1.70GHz (broadwell), Python 2.7.5
See https://gist.github.com/199b00a7bf8ab677ad8abbac0ddd4e4a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 16 out of 16 (4 easyconfigs in total)
node3101.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/5aebc59ca8acb8b9ee85b8cdf3897a29 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
jsfc061 - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, 1 x NVIDIA Tesla V100-PCIE-16GB, 470.57.02, Python 3.6.8
See https://gist.github.com/b1eb543e72fdc9c3957ee470ddcaf130 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3300.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), 1 x NVIDIA NVIDIA Tesla V100-SXM2-32GB, 465.19.01, Python 3.6.8
See https://gist.github.com/9d6377b194b5da23cbe580708bbb07f5 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @Flamefire!

**Comment by Flamefire** [comment]
Test report by @Flamefire
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
taurusml25 - Linux RHEL 7.6, POWER, 8335-GTX (power9le), Python 2.7.5
See https://gist.github.com/e8e3f3527ba9024acce80d0a02675715 for a full test report.

**Pull request merged**